test/spec/nps-form/nps-formSpec.js

```diff
@@ -18,13 +18,13 @@ describe('Unit testing nps form', function() {
     it('Contains the Net Promote Score form', function(){
         var element = $compile("<nps-form></nps-form>")($rootScope);
         $rootScope.$digest();
-        expect(element.html()).toContain('<input type="radio" name="nps" value="0">0<input type="radio" name="nps" value="1">1<input type="radio" name="nps" value="2">2<input type="radio" name="nps" value="3">3<input type="radio" name="nps" value="4">4<input type="radio" name="nps" value="5">5<input type="radio" name="nps" value="6">6<input type="radio" name="nps" value="7">7<input type="radio" name="nps" value="8">8<input type="radio" name="nps" value="9">9<input type="radio" name="nps" value="10">10');        
+        expect(element.html()).toContain(' <div name="rating" class="flex-radio">');        
```

Comment:
Since this is publicly available, we should prefix or CSS classes with something to indicate is CSS for the nps form... Perhaps nps?